core/Types.h

```diff
@@ -704,6 +704,7 @@ struct DispatchArgs {
 
     NameRef name;
     const CallLocs &locs;
+    u2 numPosArgs;
```

Comment:
Again, this is the same as `ast::Send::numPosArgs`. There's a lot of duplication of this field around, and I had considered making a template to wrap up the usage patterns. It might still be worth doing that, but it might be better left to a follow-up PR at this point.